### PR TITLE
Release Emulator UI v1.5.1.

### PR DESCRIPTION
**Pull request by yuchenshi**
### Description

See changelog

### Scenarios Tested

Started Emulator UI locally and ran through a few tabs to make sure nothing crashed. Tried some basic Storage use cases and all seems well.

### CODE REVIEW

**Review by samtstern — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 - Support loading Firebase Functions packaged as an ES module. (#3485)
 - Fixes Cloud Storage emulator missing classes. (#3541)
 - Add missing properties to Authentication functions `context` in the Functions emulator. (#3536)
+- Firestore Emulator UI now supports deleting documents and collections recursively.
+- Fixes some Storage Emulator UI errors.
```
Comment (samtstern) [inline CHANGELOG.md:5]: Any links to issues for "some errors"?

**Review by samtstern — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (yuchenshi) [reply CHANGELOG.md:5]: Issues are on a different repo, such as https://github.com/firebase/firebase-tools-ui/issues/576. I don't know the right convention for that.

Comment (samtstern) [reply CHANGELOG.md:5]: I think just linking them like that works, GitHub makes it pretty nice and short. But not a blocker.

Comment (yuchenshi) [reply CHANGELOG.md:5]: Just tried and the nice and short formatting doesn't work in regular Markdown files. Just comments.

**Pull request merged**